releasenotes/README.md

```diff
@@ -26,12 +26,39 @@ Additional documentation:
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
 
-## 2.14.6 (Oct 25, 2023) [stable-rc]
+## 2.15.7 (Nov 1, 2023) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
+* NEW: The `dev convert script` command for migrating your CI scripts to use the `sf`-style commands now handles `sfdx` commands that span multiple lines.  For example, the command converts this:
+
+    ```bash
+    sfdx force:user:create -f config/user-def.json \
+    -a myuser \
+    -u myorg
+    ```
+    into this:
+
+    ```bash
+    sf org create user --definition-file config/user-def.json \
+    --set-alias myuser \
+    --target-org myorg
+    ```
+    (plugin-dev PR [#402](https://github.com/salesforcecli/plugin-dev/pull/402))
+  
+* FIX: The `org list` command now correctly renders the output when a listed scratch org has expired. (GitHub issue [#2533](https://github.com/forcedotcom/cli/issues/2533), plugin-org PR [#845](https://github.com/salesforcecli/plugin-org/pull/845))
```

Comment:
the fix is for non-scratch/hub orgs. Like a deactivated/expired developer edition org
https://help.salesforce.com/s/articleView?id=sf.admin_deactivate_org.htm&type=5

Reply:
Good catch!  I'll update it, thanks.